internal/validator/validator_test.go

```diff
@@ -0,0 +1,51 @@
+package validator
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidator_ValidEmails(t *testing.T) {
```

Comment:
Nice, you have table tests!

Reply:
tiny: I think these tests can be merged into one. Than test case struct will look like
```
{
  name  string
  email string
  isValid bool
  expectedErrorMessage string
}
```

Reply:
Resolved with [c4f7ffb](https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-fdemchenko/pull/4/commits/c4f7ffb0095e9a0b4768fbf18e85014cd109b08a)